tests/support_server/setup.pm

```diff
@@ -283,6 +280,98 @@ sub setup_xdmcp_server {
     $xdmcp_server_set = 1;
 }
 
+sub setup_iscsi_server {
+    return if $iscsi_server_set;
+
+    # If no LUN number is specified we must die!
+    my $num_luns = get_required_var('NUMLUNS');
+
+    # A second disk for iSCSI LUN is mandatory
+    my $hdd_lun_size = get_required_var('HDDSIZEGB_2');
+
+    # Integer part of the LUN size is keep and can't be lesser than 1GB
+    my $lun_size = int($hdd_lun_size / $num_luns);
+    die "iSCSI LUN cannot be lesser than 1GB!" if ($lun_size < 1);
+
+    # Are we using virtio or virtio-scsi?
+    # Note: it should be possible to do this in pure Perl code...
+    my $hdd_lun = script_output "
+    for DEV in vdb sdb; do
+      ls /dev/\${DEV} >/dev/null 2>&1     \\
+        && echo \${DEV} > /dev/$serialdev \\
+        && break
+    done";
+    die "detection of disk for iSCSI LUN failed" unless $hdd_lun;
+
+    # Needed if a firewall is configured
+    script_run 'yast2 firewall services add zone=EXT service=service:target';
+
+    # Create the iSCSI LUN
+    script_run "parted --align optimal --wipesignatures --script /dev/$hdd_lun mklabel gpt";
+    my $start = 0;
+    my $size  = 0;
+    for (my $num_lun = 1; $num_lun <= $num_luns; $num_lun++) {
+        $start = $size + 1;
+        $size  = $num_lun * $lun_size * 1024;
+        script_run "parted --script /dev/$hdd_lun mkpart primary ${start}MiB ${size}MiB";
+    }
+
+    # The easiest way (really!?) to configure LIO is with YaST
+    # Code grab and adapted from tests/iscsi/iscsi_server.pm
+    script_run("yast2 iscsi-lio-server; echo yast2-iscsi-lio-server-status-\$? > /dev/$serialdev", 0);
+    assert_screen 'iscsi-lio-server';
+    assert_screen 'iscsi-target-overview-service-tab';
```

Comment:
Why two assert_screen without action between them? Could be replaced with better needle maybe